emperor/_pandas.py

```diff
@@ -0,0 +1,102 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, emperor development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.md, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+import numpy as np
+import pandas as pd
+
+from emperor.core import Emperor
+from skbio import OrdinationResults
+
+
+def scatterplot(df, x=None, y=None, z=None, remote=True):
+    """Create an Emperor scatter plot from a Pandas DataFrame
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Pandas DataFrame with the data to display
```

Comment:
Great catch!

On (Jul-11-16|12:08), Jamie Morton wrote:
> > +from __future__ import division
> > +
> > +import numpy as np
> > +import pandas as pd
> > +
> > +from emperor.core import Emperor
> > +from skbio import OrdinationResults
> > +
> > +
> > +def scatterplot(df, x=None, y=None, z=None, remote=True):
> > +    """Create an Emperor scatter plot from a Pandas DataFrame
> > +
> > +    Parameters
> > +    ----------
> > +    df : pd.DataFrame
> > +        Pandas DataFrame with the data to display
> 
> Don't forget to note that this also includes metadata as well as the PC coordinates.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/517/files/47d735bcbfbd8fc467948a139261bdf5a7092d82#r70317772